docs/source/multithreading.rst

```diff
@@ -33,6 +33,7 @@ The following is a list of behaviors to keep in mind when using control flow fun
 
 - `cont` will continue all threads.
 - `step` and `step_until` will step the selected thread.
+- `next` will step on the selected thread or, if a call function is found, continue on all threads until the end of the function or another stopping event.
```

Comment:
*called function

Reply:
Resolved in [fb11af7](https://github.com/libdebug/libdebug/pull/80/commits/fb11af7cf439a12391ae1831aee03fcd2b39b45f)